sycl/test/basic_tests/types.cpp

```diff
@@ -1,4 +1,4 @@
-// RUN: %clangxx -fsycl %s -o %t.out
+// RUN: %clangxx -fsycl %s -o %t.out -lOpenCL
```

Comment:
What is `-lOpenCL` added for?

Reply:
Remained from past patch. I take it we don't write it anymore. Removed.